### DISCUSSION TITLE
Does Fluent Assertions change the build of projects that use it?

### DISCUSSION THREAD

**Discussion by cremor** [fluentassertions/fluentassertions#2139] [Q&A] [answered]
I just found this issue in the library "Shouldly" which has similar goals as  Fluent Assertions : https://github.com/shouldly/shouldly/issues/795

Now I'm wondering: Does  Fluent Assertions also change how projects that reference it are build?

**Comment by dennisdoomen** [comment, marked as answer]
It doesn't. But it needs the same changes to the test project that Shouldly needs to be able to [identify the caller](https://fluentassertions.com/introduction#subject-identification).
